pmm-app/src/pmm-dbaas/components/DBCluster/AddDBClusterModal/DBClusterBasicOptions/DBClusterBasicOptions.tsx

```diff
@@ -21,6 +24,62 @@ export const DBClusterBasicOptions: FC<DBClusterBasicOptionsProps> = ({ kubernet
     change(AddDBClusterFields.databaseType, databaseType);
   }, []);
 
+  const kubernetesOptions = kubernetes.map((kubernetesCluster) => {
+    const { kubernetesClusterName, operators } = kubernetesCluster;
+
+    const availableOperators = OPERATORS.filter(
+      (databaseType) => operators[databaseType].status === KubernetesOperatorStatus.ok,
+    );
+    const disabledOperators = OPERATORS.filter(
+      (databaseType) => operators[databaseType].status !== KubernetesOperatorStatus.ok,
+    );
+
+    const getOptionContent = (listOfOperators, kubernetesClusterName) => (
+      <OptionContent
+        title={kubernetesClusterName}
+        description={
+          disabledOperators.length ? Messages.dbcluster.addModal.validationMessages.notInstalledOperator : ''
+        }
+        tags={availableOperators.map((databaseType) => DatabaseOperators[databaseType])}
+        disabledTags={disabledOperators.map((databaseType) => DatabaseOperators[databaseType])}
+      />
+    );
+
+    return {
+      value: kubernetesClusterName,
+      label: getOptionContent(OPERATORS, kubernetesClusterName),
+      operators,
+    };
+  });
+
+  const [databaseOptions, setDatabaseOptions] = useState(DATABASE_OPTIONS);
+  const onChangeCluster = useCallback((selectedKubernetes) => {
+    const { operators } = selectedKubernetes;
+    const availableDatabaseOptions: DatabaseOption[] = [];
+
+    if (operators.xtradb.status === KubernetesOperatorStatus.ok) {
+      availableDatabaseOptions.push({
+        value: Databases.mysql,
+        label: DATABASE_LABELS[Databases.mysql],
+      });
+    }
+
+    if (operators.psmdb.status === KubernetesOperatorStatus.ok) {
+      availableDatabaseOptions.push({
+        value: Databases.mongodb,
+        label: DATABASE_LABELS[Databases.mongodb],
+      });
+    }
+
+    change(AddDBClusterFields.databaseType, {
```

Comment:
Why put the value for this field to undefined? Isn't it undefined by default?

Reply:
It's, but we need to do it to clean the database type field after k8s cluster selection